drivers/fastboot/fb_mmc.c

```diff
@@ -567,9 +566,11 @@ void fastboot_lock_critical(char *cmd_parameter, char *response)
  */
 void fastboot_unlock_critical(char *cmd_parameter, char *response)
 {
-	u8 ack = 1;
-	u8 part_num = 1;
-	u8 access = 1;
+	#define BOOT_ACK_EN		1
+	#define BOOT_PART_ACCESS	CONFIG_SYS_MMC_ENV_PART;
+	u8 ack = BOOT_ACK_EN;
+	u8 part_num = CONFIG_SYS_MMC_ENV_PART;
+	u8 access = BOOT_PART_ACCESS;
 	int ret = -1;
```

Comment:
Since this patch only modifies the function added in the previous one, please squash them both together.
Thanks!